tests/e2e/utils/requestHandlers/MultiUserHeaderHandler.ts

```diff
@@ -0,0 +1,28 @@
+/*********************************************************************
+ * Copyright (c) 2019 Red Hat, Inc.
+ *
+ * This program and the accompanying materials are made
+ * available under the terms of the Eclipse Public License 2.0
+ * which is available at https://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ **********************************************************************/
+import { AxiosRequestConfig } from 'axios';
+
+import { CLASSES } from '../../inversify.types';
+import { IHeaderHandler } from './IHeaderHandler';
+import { injectable, inject } from 'inversify';
+import { TokenHandler } from '../TokenHandler';
+
+@injectable()
+export class MultiUserHeaderHandler implements IHeaderHandler {
+
+    constructor(@inject(CLASSES.TokenHandler) private readonly tokenHandler: TokenHandler) {
+    }
+    async getHeaders() : Promise<AxiosRequestConfig> {
+        let token = await this.tokenHandler.getCheBearerToken();
```

Comment:
According to the best practices, we should use "const" where it possible.
In this case, ```token``` will not be changed so better to use ```const``` instead ```let```

Reply:
Also, it would be great don't forget about types, so declare the "string" type here, please.